core/src/main/scala/org/apache/spark/scheduler/TaskDescription.scala

```diff
@@ -57,7 +56,10 @@ private[spark] class TaskDescription(
     val artifacts: JobArtifactSet,
     val properties: Properties,
     val cpus: Int,
-    val resources: immutable.Map[String, ResourceInformation],
+    // resources is the total resources assigned to the task
+    // Eg, Map("gpu" -> Map("0" -> ResourceAmountUtils.toInternalResource(0.7))):
+    // assign 0.7 of the gpu address "0" to this task
+    val resources: immutable.Map[String, immutable.Map[String, Long]],
```

Comment:
nit: Shall we create a named type for this map and have its key and value better clarified? @wbo4958

Reply:
Do you think we should have a new PR for master branch for your comment? @Ngone51